assignments/objects.js

```diff
@@ -19,26 +19,75 @@ const example = {
 
 // Write your intern objects here:
 
+const mitzi = {
+  id: 1,
+  name: "Mitzi",
+  email: "mmelloy0@psu.edu",
+  gender: "F"
+}
+
+const kennan = {
+  id: 2,
+  name: "Kennan",
+  email: "kdiben1@tinypic.com",
+  gender: "M"
```

Comment:
so I would have put my speak functin for kennan directly inside his object:

const kennan = {
  id: 2,
  name: "Kennan",
  email: "kdiben1@tinypic.com"
  gender: "M",
  speak: function(arg) {
     console.log(`Hello, my name is ${this.name}`)
  },
}

Notice that it grabs the name by referencing it by this.name.  You will learn more about this later.